src/lib/core/option/option.ts

```diff
@@ -99,17 +92,22 @@ export class MdOption extends _MdOptionMixinBase implements CanDisableRipple {
   get disabled() { return (this.group && this.group.disabled) || this._disabled; }
   set disabled(value: any) { this._disabled = coerceBooleanProperty(value); }
 
+  /** Whether ripples for the option are disabled. */
+  @Input()
```

Comment:
It might be true that it might not happen that someone just disables ripples for a single option.

From the payload size perspective it shouldn't make any difference, so it's more a question of API design.

I'm fine removing the `@Input` here. Also since https://github.com/angular/material2/commit/addf1ce8b124fe7a242bf927e59ea69d00513f85 is not in `master` that long, I assume it won't be a breaking change.